tools/test/js/app.js

```diff
@@ -101,7 +101,7 @@ function IframeComponent({
       <iframe
         ref={iframeRef}
         id={id}
-        width="300px"
+        width={userId === "1" ? "630px" : "300px"}
```

Comment:
why userId === "1" ??

Reply:
The size was divided for verification when the width changed, so it is unnecessary so I will delete it.